keyboards/sofle_pico/keymaps/default/keymap.c

```diff
@@ -0,0 +1,161 @@
+        _______, KC_LALT, KC_LCTL , KC_LSFT , XXXXXXX , KC_CAPS ,                          KC_PGDN, KC_LEFT, KC_DOWN, KC_RGHT, KC_DEL , KC_BSPC,
+        _______, KC_UNDO, KC_CUT  , KC_COPY , KC_PASTE, XXXXXXX , _______,        _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, _______,
+                          _______ , _______ , _______ , _______ , _______,        _______, _______, _______, _______, _______
+        ),
+        /* ADJUST
+        * ,-----------------------------------------.                    ,-----------------------------------------.
+        * |      |      |      |      |      |      |                    |      |      |      |      |      |      |
+        * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
+        * |QK_BOOT|     |      |      |      |      |                    |      |      |      |      |      |      |
+        * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
+        * |      |      |MACWIN|      |      |      |-------.    ,-------|      | VOLDO| MUTE | VOLUP|      |      |
+        * |------+------+------+------+------+------|  MUTE |    |       |------+------+------+------+------+------|
+        * |      |      |      |      |      |      |-------|    |-------|      | PREV | PLAY | NEXT |      |      |
+        * `-----------------------------------------/       /     \      \-----------------------------------------'
+        *            | LGUI | LAlt | LCTR |LOWER | /Enter  /       \Space \  |RAISE | RCTR | RAlt | RGUI |
+        *            |      |      |      |      |/       /         \      \ |      |      |      |      |
+        *            `----------------------------------'           '------''---------------------------'
+        */
+        [_ADJUST] = LAYOUT(
+        XXXXXXX , XXXXXXX, XXXXXXX , XXXXXXX, XXXXXXX, XXXXXXX,                       XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+        QK_BOOT , XXXXXXX, XXXXXXX , XXXXXXX, CG_TOGG, XXXXXXX,                       XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+        XXXXXXX , XXXXXXX, CG_TOGG , XXXXXXX, XXXXXXX, XXXXXXX,                       XXXXXXX, KC_VOLD, KC_MUTE, KC_VOLU, XXXXXXX, XXXXXXX,
+        XXXXXXX , XXXXXXX, XXXXXXX , XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,     XXXXXXX, XXXXXXX, KC_MPRV, KC_MPLY, KC_MNXT, XXXXXXX, XXXXXXX,
+                            _______, _______, _______, _______, _______,     _______, _______, _______, _______, _______
+        )
+    // clang-format on
+};
+
+/* Default timeout for displaying logo on boot. */
+#ifndef OLED_LOGO_TIMEOUT
+/* 10 second timeout. */
+#    define OLED_LOGO_TIMEOUT 10000
+#endif
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+    return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
```

Comment:
May want to use the tri layer feature, instead: 
https://docs.qmk.fm/#/feature_tri_layer

Reply:
@drashna Excellent suggestion! 
I revised to include tri layer. 
Is `TRI_LAYER_ENABLE = yes` still required in `rules.mk`? I didn't see it in all of the boards that are using `TL_LOWR` & `TL_UPPR`?

Reply:
Via enables it by default.  Along with a few other features.

But non-via keymaps (eg default) would need it explicitly enabled.

Reply:
Thanks! I pulled it out of the VIA `rules.mk`, and left it in the default keymap `rules.mk`